synapse/tests/test_lib_layer.py

```diff
@@ -1388,3 +1388,44 @@ async def test_layer_setinfo(self):
 
             with self.raises(s_exc.BadOptValu):
                 await core.callStorm('$layer = $lib.layer.get() $layer.set(newp, hehe)')
+
+    async def test_reindex_byarray(self):
+
+        async with self.getRegrCore('reindex-byarray') as core:
+
+            layr = core.getView().layers[0]
+
+            nodes = await core.nodes('transport:air:flightnum:stops*[=stop1]')
```

Comment:
do we have test forms which are a lil more terse that can be used here? ``test:arrayprop`` ?